### ISSUE TITLE
Change spokesman (on /) to spokesperson and buzzword (on /specs/) to obsolete

### ISSUE THREAD

**Issue by annevk** [whatwg/whatwg.org#15]

**Comment by zcorpan** [comment]
Maybe instead of saying "obsolete "HTML5"" we can rephrase it to say what we think the technologies actually are and avoid talking about "HTML5"

**Comment by zcorpan** [comment]
The WHATWG works on a number of technologies that are fundamental parts of the web platform.

(While here we can un-capitalize "Web")
